lib/chat_app/application.ex

```diff
@@ -5,6 +5,16 @@ defmodule ChatApp.Application do
 
   use Application
 
+  @impl true
+  def start_phase(:tables, :normal, _) do
+    ChatApp.ETSDatabase.initialize() |>
+    Enum.map(fn row ->
+      :ets.new(elem(row, 0), elem(row, 1))
+    end)
```

Comment:
Why not pattern match against the `row` tuple?
Instead of:
```elixir
fn row -> call_something(elem(row, 0), elem(row,1)) end
```

try:
```elixir
fn {name, properties} -> call_something(name, properties) end
```
since it's easier to read.